src/theory/quantifiers/sygus/sygus_grammar_cons.cpp

```diff
@@ -1079,6 +1085,20 @@ void CegGrammarConstructor::mkSygusDefaultGrammar(
       // This ensures that ( c1*x + c2*y >= 0 ) has the same weight as
       // e.g. ( x >= 0 ) or ( y >= 0 ).
       sdts[iat].d_sdt.addConstructor(op, "polynomial", cargsAnyTerm, spc, 0);
+      // define zero integer dtype
```

Comment:
Does this have to be a zero type?  I thought of doing this but I think we can just build `0` as part of the sygus operator for the Boolean.  As in, its sygus constructor becomes (lambda cx cy.  (cx*x + cy*y >= 0) instead of (lambda cx cy czero.  cx*x + cy*y >= czero).

Reply:
You know, I spent a ridiculous amount of time remembering and getting right the thing of adding new types / datatypes / initializing them properly and then as I submitted I thought "maybe I could just have done a lambda"? You're right, I'll fix.

Reply:
You know, I spent a ridiculous amount of time remembering and getting right the thing of adding new types / datatypes / initializing them properly and then as I submitted I thought "maybe I could just have done a lambda"? You're right, I'll fix.

Reply:
Haha I know the feeling :)